configure.ac

```diff
@@ -1955,6 +1955,13 @@ if test "x$mono_feature_disable_gac" = "xyes"; then
 	AC_MSG_NOTICE([Disabled GAC support])
 fi
 
+# Currently, we only want to use DllMap for non-netcore builds
+if test "x$with_core" = "xno" -a "x$mono_feature_disable_dllmap" != "xyes"; then
+    AC_DEFINE(ENABLE_DLLMAP,1,[Enables use of DllMaps in MonoVM])
+    with_dllmap=yes
+fi
+AM_CONDITIONAL(ENABLE_DLLMAP, [test "x$with_dllmap" = "xyes"])
+
```

Comment:
Is this automake conditional needed ?

Reply:
Not currently, but I added it originally to make it available if needed down the line. If you'd prefer without that's an easy change.

Reply:
Removed.